.github/workflows/ci.yml

```diff
@@ -226,6 +226,71 @@ jobs:
           npm install -g markdownlint-cli@0.31.1
           markdownlint --config .markdownlintrc **/*.md docs/src/ 
 
+  publish-docker-image:
+    needs: cargo-toml-fmt-check
+
+    if: always()
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      # This is a way to make this job run after publish-crates even if it's skipped on master or pr branches
+      # https://stackoverflow.com/a/69252812/680811
+      - name: fail if any dependent jobs failed
+        if: ${{ contains(needs.*.result, 'failure') }}
+        run: exit 1
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/fuellabs/fuel-core
```

Comment:
@ra0x3 - you wanna change this to gchr.io repo for fuel-indexer if you intend to create the image e.g. ghcr.io/fuellabs/fuel-indexert